yolact.py

```diff
@@ -19,12 +19,14 @@
 
 # This is required for Pytorch 1.0.1 on Windows to initialize Cuda on some driver versions.
 # See the bug report here: https://github.com/pytorch/pytorch/issues/17108
+from utils.sparse import SparseMLWrapper
+
 torch.cuda.current_device()
 
 # As of March 10, 2019, Pytorch DataParallel still doesn't support JIT Script Modules
-use_jit = torch.cuda.device_count() <= 1
+use_jit = False
 if not use_jit:
-    print('Multiple GPUs detected! Turning off JIT.')
+    print('JIT turned off')
```

Comment:
let's remove this log since use_jit is now always False.  Add a comment saying that this is disabled for the SparesML integration due to torch quantization compatibility